src/item.cpp

```diff
@@ -3983,10 +3966,10 @@ long item::ammo_required() const
     return 0;
 }
 
-bool item::ammo_consume( int qty, const tripoint& pos ) {
+long item::ammo_consume( long qty, const tripoint *pos ) {
```

Comment:
> Or just always require the position to be specified.

The latter isn't always possible

> Then add is_null to tripoint and have it internally compare itself to tripoint_min

That's more complex than depending on `nullptr` meaning `null`